scripts/scrape_issue_info.py

```diff
@@ -28,11 +31,19 @@ def get_issue_hash(issue_num: str, token: str):
     }
     r = requests.get(issue_url, headers=params)
     response = json.loads(r.text)
-    print(response["title"].split(" ")[-1])
+    return response
 
 def main():
     args = parse_arguments()
-    issue_hash = get_issue_hash(args.num, args.token)
+    issue = get_issue(args.num, args.token)
+    issue_hash = issue["title"].split(" ")[-1]
+    print(issue_hash)
+    m = re.findall('\[[^]]*\]', issue['body'])
+    count = Counter(m)
+    with open("bisection_baseline.txt", "w") as f:
+        most_common = count.most_common(1)[0][0][1:-1]
```

Comment:
Can you explain what's going on here? I'm not following this block of code with the counter(m).
Also IIUC you can move this statement outside of the file open context.

Reply:
> Can you explain what's going on here? I'm not following this block of code with the counter(m).  

This is for bisections which means we have the issue number. From the body markdown, we search for previous hashes with the regex matching all values inside of `[]`. These are potential previous hashes (since if one target failed to build previously but built this time, the prev hash of that target may be different). Count the most common occurrences of the previous hash (using counter) and use that as our baseline bisection hash.
> Also IIUC you can move this statement outside of the file open context.

that I can